packages/react-theme/src/utils/mergeThemes.ts

```diff
@@ -2,5 +2,5 @@ import { merge } from '@fluentui/utilities';
 import { PartialTheme, Theme } from '../types';
 
 export function mergeThemes(a: Theme, b: PartialTheme | Theme): Theme {
-  return merge(a, b) as Theme;
+  return merge({}, a, b) as Theme;
```

Comment:
A small bugfix, `merge()` mutates the first object 😨

Reply:
can this be covered in unit tests?

Reply:
Oops, overlooked your comment, yes it can be